lib/src/session.dart

```diff
@@ -131,7 +136,8 @@ class Session with ChangeNotifier {
       );
     }
 
-    if ((unreads != null) && (unreads!.onChange != null)) {
+    if ((onUnreadsChange != null) ||
+        ((unreads != null) && (unreads!.onChange != null))) {
```

Comment:
We can simplify this to:
```suggestion
    if ((onUnreadsChange != null) || (unreads?.onChange != null)) {
```

Reply:
Done👍